### PR TITLE
[REVIEW] Notices for 21.10 and 21.12

### PR DESCRIPTION
**Pull request by Salonijain27**

### CODE REVIEW

**Comment by Salonijain27** [comment]
@fondaing and @raydouglass please can I get a review

File: _notices/rsn0011.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+notice_updated: 2021-10-12
+---
+
+## Overview
+
+We are increasing the minimum driver version for RAPIDS CUDA 11.0 to 450.80.02+ in order to support CUDA Enhanced Compatibility 
+
+## Impact
+
+Users would have to install CUDA driver 450.80.02+ in order to use CUDA 11.0
```
Comment (raydouglass) [inline _notices/rsn0011.md:33]: I would say something like: Users should upgrade their driver to `450.80.02` or newer to continue using RAPIDS in the future

File: _notices/rsn0012.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 12 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "SQL migration in 21.12"
```
Comment (raydouglass) [inline _notices/rsn0012.md:10]: Should be `v21.12` for consistency

File: _notices/rsn0012.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+notice_updated: 2021-10-13
+---
+
+## Overview
+
+RAPIDS will preview its migration path from BlazingSQL to dask-based SQL in 21.12
+
+## Impact
+
+21.12 will contain a preview of a migration path for users to refer to use RAPIDS with dask-based SQL
```
Comment (raydouglass) [inline _notices/rsn0012.md:33]: I think the words `to refer` can be removed

**Review by raydouglass — APPROVED** [review]

**Pull request merged**